connector/datadogconnector/connector.go

```diff
@@ -46,6 +52,13 @@ type traceToMetricConnector struct {
 
 var _ component.Component = (*traceToMetricConnector)(nil) // testing that the connectorImp properly implements the type Component interface
 
+// cacheExpiration is the time after which a container tag cache entry will expire
+// and be removed from the cache.
+var cacheExpiration = time.Minute * 5
```

Comment:
Do we want to make this more configurable?

Reply:
This is temporary. We should get rid of it next quarter, where concentrator picks up tags from the incoming payload.